webboot/webboot.go

```diff
@@ -178,6 +178,19 @@ func usage() {
 func main() {
 	flag.Parse()
 
+	commandline, err := ioutil.ReadFile("/proc/cmdline")
+
+
+	if err != nil {
+		fmt.Errorf("Unable to read /proc/cmdline: %v", err)
```

Comment:
This line has no effect. Just do this:
log.Fatal(err)
The error message will have that file name in it.
https://play.golang.org/p/xrfTNOGuzBM